docs/unit-testing.md

```diff
@@ -113,10 +112,3 @@ Sample output:
 docker run --rm -it devtestdemisto/python:1.3-alpine-1b9f5bee16a24c3f5463e324c1bb075e sh
 ```
 
-* If you have faced the error `ValueError: unknown locale: UTF-8` when running `pkg_dev_test_tasks.py`, add these lines to your ~/.bash_profile: 
-```
-export LC_ALL=en_US.UTF-8
-export LANG=en_US.UTF-8
```

Comment:
Why did you delete this? Let's leave this trouble shooting tip. Just change `pkg_dev_test_tasks.py` to `demisto-sdk`.